db/migrate/20220712104948_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
- [ ] Kindly, make sure that wherever you have `text`  column, you use text as type instead of using string. text data type field has more characters and can contains up to 30000 characters.